data/src/client.rs

```diff
@@ -816,6 +863,47 @@ impl Client {
                 #[cfg(feature = "dev")]
                 return None;
             }
+            Command::Numeric(RPL_ISUPPORT, args) => {
+                args[1..].iter().for_each(|arg| {
+                    if arg != "are supported by this server" {
```

Comment:
I don't think this is valid since the text may be something else (see @casperstorm comment).

We should skip the _last_ arg if it's a non-valid parameter.

Reply:
https://github.com/squidowl/halloy/pull/349#issuecomment-2073438081